### ISSUE TITLE
APU On/Off changes status without power 

### ISSUE THREAD

**Issue by tluellma** [Octal450/MD-11#3]
This is a very small one.

After pressing "Battery" in "Cold and dark" you may press the APU button located between all these fuel pump switches. Nothing seems to happen.

Steps to reproduce the behaviour:
1. Start "Cold and dark"
2. Press Battery button
3. Press APU button (the one between all the fuel pumps)
4. Nothing seems to happen.
5. Now start APU with APU start button (at the very top right of the OH Panel)
6. Wait for APU Power to become "Avail"
7. Now press the APU Power Switch which is displaying "Avail"
8. You see, that the button from Position 3 has state "On" now.

9. Repeat steps 1,2, skip 3 and 4, 5, 6, 7

Now the APU button from step 3 is still dark which would be expected behaviour to me.

** System (please complete the following information):**
 - OS: Windows 10
 - FlightGear version: 2020.1.1

**Comment by Octal450** [comment]
Sorry, I'm a little confused. There is no APU button on the fuel panel, do you mean the electrical panel? If so, I am aware that that button doesn't start the APU yet, that will be added when I overhaul the APU system (on the list after engine startup)

Here's what happens for me:

By step 6 I get green AVAIL, and if I then press the APU Power Button on the electric panel, I get blue ON. 

After doing step 1 and 2 again, the APU Power switch and the APU start switch are both dark as expected as the APU has not been started.

Could you possibly perform a video of this?

Kind Regards,
Josh

**Comment by tluellma** [comment]
[

](url)

**Comment by tluellma** [comment]
I tried to make a video, but it is too big to upload. 

Let me try to explain again:

There are three buttons for the APU. One is the "APU Master Button" in the very top right of the OH-Panel.

One is in the electric panel and one is in the AIR panel (not Fuel Panel - I was wrong here).

If I start the Battery Master and then the "APU Master Button" after a while the APU Button in the Electric Panel will show "Avail" in green colour. If I press that one, the aircraft will get some electric power. Note that the APU button in the AIR Section (below the Electricity button right below the "3-ISO"-Button) stays black. That is expected behaviour for me.

Now Cold and dark again.

Battery Master switch to on. (Emergency Power still OFF). Now click one time on the APU Button below the 3-ISO-Button in the AIR Panel. Nothing seems to happen now. Do not click it twice.

Now start the APU with the "APU Master Button" in the top. When the electric APU Button shows "Avail" you are able to put some electricity into the aircraft. Press it now and power it up.

Now you will see that the APU "Air" button is already set to "On". It remembers your mouse click even if there was no power at that time. 

It is a very small issue. I think that kind of switch needs some power to store a state.

I hope that cleared it a bit up.

**Comment by Octal450** [comment]
Ahh ok, I wasn't aware that you talked about the APU Bleed button. I will attempt to reproduce this later.

Thank you!
Josh

**Comment by Octal450** [comment]
Hi,
I just checked it and its not wrong.
According to the electrical schematic, most of the overhead buttons are switched to the DC Battery Bus, but most of the lamps are powered only from the Main DC Busses, and not the DC Battery Bus.
The idea being, that you can restore power via the ADG or not via electrical - so you need those lamps, but for the rest you can just check the configuration on the systems display synoptic, by calling it up (i believe it shows up on the EAD) and then dismiss it when done.

It will not change state if the power is not on the DC Bat Bus or the Main DC Busses.

Thank you though :)

Kind Regards,
Josh
